test/functional/inspec_exec_test.rb

```diff
@@ -46,6 +46,15 @@
     File.stat("#{outpath}/foo/bar/test.json").size.must_be :>, 0
   end
 
+  it 'can execute the profile with a target_id passthrough' do
+    outpath = Dir.tmpdir
+    out = inspec("exec #{example_profile} --no-create-lockfile --target-id 1d3e399f-4d71-4863-ac54-84d437fbc444")
+    out.stderr.must_equal ''
+    out.exit_status.must_equal 101
+    stdout = out.stdout.force_encoding(Encoding::UTF_8)
+    stdout.must_include "ID:      1d3e399f-4d71-4863-ac54-84d437fbc444"
```

Comment:
Minor gripe, but if we're asking for the user to pass this using the `--target-id` option, I think we should consistently refer to it as a *target* id.  We may have other ID's in the future - report IDs, etc.

Reply:
Actually I think I may leave this for now. The other formatting options look bad to me: